skimage/filters/setup.py

```diff
@@ -33,6 +34,12 @@ def configuration(parent_package='', top_path=None):
         'rank.bilateral_cy', sources=['rank/bilateral_cy.c'],
         include_dirs=[get_numpy_include_dirs()])
 
+    ext = pythran.dist.PythranExtension(
+        'skimage.filters._diffusion_utils_pythran',
+        sources=["skimage/filters/_diffusion_utils.py"],
```

Comment:
I was initially confused about which module is Pythran is applied to. How do you feel about renaming "_diffusion_utils.py" to "_diffusion_utils_pythran.py" to indicate that it's compiled? That would also name the output files more consistent.